### ISSUE TITLE
"mark blank" should be logged as a version (?)

### ISSUE THREAD

**Issue by saracarl** [benwbrum/fromthepage#4046]
Should we note a "marked blank" as a version?  Seems more relevant in scenarios where there is (bad/hallucinated) AI text or OCR text.

**Comment by benwbrum** [comment]
Yes.  We should create a new page version to record the new blank status, and handle the display of the version as if the source text had been deleted (unless, of course, we actually do delete the text -- I don't quite recall how our code handles the situation of someone marking a page blank that previously had transcribed text).

**Comment by benwbrum** [comment]
According to #5477 we do not currently blank out the transcription when the user presses 'mark' blank.  If we did (as implemented in #5477), the page version would be logged.  However, there is a reason we don't blank out the text -- we don't want users to lose it if they accidentally marked a page blank.

This feature needs more definition.
